Packs/Carbon_Black_Enterprise_Response/ReleaseNotes/2_1_42.md

```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Block File - Carbon Black Response
+
+- The playbook is now using the integration 'VMware Carbon Black EDR v2' that replaced the EOL integration 'VMware Carbon Black EDR v2'.
```

Comment:
Do you mean:
The playbook is now using the integration 'VMware Carbon Black EDR v2' that replaced the EOL integration 'VMware Carbon Black EDR'.